eyeshade/workers/acvote.js

```diff
@@ -0,0 +1,73 @@
+const { votesId } = require('../lib/queries.js')
+const { voteType } = require('../lib/vote.js')
+
+const voteTopic = process.env.ENV + '.payment.vote'
+
+module.exports = (runtime, callback) => {
+  runtime.kafka.on(voteTopic, async (messages) => {
+    const client = await runtime.postgres.connect()
+    const now = new Date()
+    const date = [
```

Comment:
The JS Date library has a lot of weirdness, especially around timezones is there any worry here? `Moment.js` might be a better option? Thoughts?

Reply:
https://github.com/brave-intl/bat-ledger/pull/822/commits/8264f629ed6b80b9459222d3da822b21226ca11b <- addressed here, also updated suggestions to match.